content/id/docs/concepts/containers/container-environment-variables.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Variabel Environment Kontainer
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+
+Laman ini menjelaskan berbagai *resource* yang tersedia di dalam Kontainer pada suatu *environment*.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## *Environment* Kontainer
+
+*Environment* Kontainer pada Kubernetes menyediakan beberapa *resource* penting yang tersedia di dalam Kontainer:
```

Comment:
Still wondering if we're mentioning something like Pod, Ingress etc should we capitalize it or not?

Reply:
IMO this is not easy to standardize -- since the upstream EN also not consistent, we'll write styleguide (like the [english](https://kubernetes.io/docs/contribute/style/style-guide/) one) after we learn probably 4-5 weeks?